src/adapters/bidderFactory.js

```diff
@@ -419,7 +419,10 @@ export function getIabSubCategory(bidderCode, category) {
 
 // check that the bid has a width and height set
 function validBidSize(adUnitCode, bid, bidRequests) {
-  if ((bid.width || bid.width === 0) && (bid.height || bid.height === 0)) {
+  bid.width = parseInt(bid.width, 10) || 0;
+  bid.height = parseInt(bid.height, 10) || 0;
+
+  if (bid.width && bid.height) {
```

Comment:
there's a subtle change here. If either `width` or `height` are 0 here then we will not return true. I don't know if we want to do that or not but it's a change in the previous behavior.

Reply:
OK, will fix.